conda_build/index.py

```diff
@@ -937,7 +939,7 @@ def index_subdir(self, subdir, index_file=None, verbose=False, progress=False):
             new_repodata_packages = {k: v for k, v in old_repodata.get('packages', {}).items() if k in unchanged_set}
             new_repodata_conda_packages = {k: v for k, v in old_repodata.get('packages.conda', {}).items() if k in unchanged_set}
 
-            for k in unchanged_set:
+            for k in sorted(unchanged_set):
```

Comment:
unknown whether sort matters, but was sorted before

Reply:
Not needed - `k` is only referenced twice elsewhere (see my other comment on that above).

Reply:
it makes for a more pleasing 'we are processing package a, then package b' instead of 'we are processing packages in random order'